_posts/2023-12-18-homebrew-4.2.0.md

```diff
@@ -0,0 +1,298 @@
+---
+title: 4.2.0
+author: MikeMcQuaid
+redirect_from: /blog/4.2.0/
+---
+
+Today, I'd like to announce Homebrew 4.2.0. The most significant changes since 4.1.0 are .
+
+Major changes and deprecations since 4.1.0:
+
+- Ruby 3
+- [Portable Ruby 3.1.4 (opt-in beta)](https://github.com/Homebrew/brew/pull/16145)
+- [Require Ruby 3.1](https://github.com/Homebrew/brew/pull/16294)
+- [Support system Ruby 3 on Linux](https://github.com/Homebrew/brew/pull/16192)
+
+- [Enable Ruby 3 for all users](https://github.com/Homebrew/brew/pull/16268)
+- [Deprecate `--new-formula`/`--new-cask` options.](https://github.com/Homebrew/brew/pull/16297)
```

Comment:
Can we expand on this slightly? The context isn't as clear here as it is in the PR itself.